examples/python/blueprint_stocks/requirements.txt

```diff
@@ -0,0 +1,3 @@
+humanize
+rerun-sdk>=0.15 # Uses the new blueprint APIs
```

Comment:
This causes conflicts when doing `pip install -r ./examples/python/requirements.txt`

```
ERROR: Cannot install rerun-sdk and rerun-sdk>=0.15 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested rerun-sdk
    The user requested rerun-sdk
    The user requested rerun-sdk>=0.15

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

pip is being pip...

Reply:
and naturally since this hasn't been uploaded yet it prevents me from insetalling the other dependencies :(

```
pip install -r ./examples/python/blueprint_stocks/requirements.txt
Collecting humanize (from -r ./examples/python/blueprint_stocks/requirements.txt (line 1))
  Using cached humanize-4.9.0-py3-none-any.whl.metadata (7.9 kB)
ERROR: Could not find a version that satisfies the requirement rerun-sdk>=0.15 (from versions: 0.2.0a2, 0.2.0a3, 0.2.0a4, 0.2.0a5, 0.2.0, 0.3.0a0, 0.3.0a1, 0.3.0a2, 0.3.0, 0.3.1, 0.4.0, 0.5.0, 0.5.1a1, 0.5.1, 0.6.0, 0.7.0, 0.8.0, 0.8.1rc1, 0.8.1, 0.8.2, 0.9.0a6, 0.9.0a9, 0.9.0a10, 0.9.0rc2, 0.9.0rc3, 0.9.0rc7, 0.9.0, 0.9.1rc3, 0.9.1rc4, 0.9.1, 0.10.0a8, 0.10.0a10, 0.10.0rc1, 0.10.0, 0.10.1rc1, 0.10.1, 0.11.0rc1, 0.11.0rc2, 0.11.0, 0.12.0a4, 0.12.0rc1, 0.12.0rc2, 0.12.0rc3, 0.12.0rc4, 0.12.0rc6, 0.12.0, 0.12.1rc1, 0.12.1rc2, 0.12.1, 0.13.0a4, 0.13.0rc1, 0.13.0rc2, 0.13.0rc3, 0.13.0rc4, 0.13.0rc5, 0.13.0rc6, 0.13.0rc7, 0.13.0, 0.14.0rc1, 0.14.0rc2, 0.14.0rc3, 0.14.0rc4, 0.14.0, 0.14.1)
ERROR: No matching distribution found for rerun-sdk>=0.15
```

Reply:
Annoying -- my mistake for expecting too much from pip.